test/inductor/test_compiled_optimizers.py

```diff
@@ -74,11 +74,11 @@ class KernelCounts(NamedTuple):
     ASGD: KernelCounts(multitensor=2, singletensor=12),
     SGD: KernelCounts(multitensor=2, singletensor=8),
     RAdam: KernelCounts(
-        multitensor=2, singletensor=None
-    ),  # Single tensor eager needs to be refactored to enable tracing (#118230)
+        multitensor=2, singletensor=12
```

Comment:
@mlazos what do the numbers here mean?

Reply:
These indicate the expected number of GPU kernels the compiler generates and launches - the single tensor case launches multiple kernels per optimizer parameter, while the multitensor version doesn't scale in this way. When compiling we expect the multitensor version to generate fewer kernels than the single tensor version